scripts/add_station.py

```diff
@@ -16,6 +16,24 @@
 
 default_coord_file_name = os.path.join(mc.data_path, 'HERA_350.txt')
 
+region = {'herahexw': [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21,
```

Comment:
These lists could be condensed a lot through judicious use of the `range()` function. Since (in Python 2) `range()` returns lists, you can concatenate them like `range(0, 47) + range(50, 61) + ...`.

Reply:
Yah, but the work already done...